polkadot-parachain/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
-name = "polkadot-parachain-bin"
-version = "0.9.300"
+name = "polkadot-parachain"
```

Comment:
@chevdor is this rollback from #1716 expected?

Reply:
nope, I brought it back here: https://github.com/paritytech/cumulus/pull/1869